### ISSUE TITLE
Incorrect path for WinKeyServer in Electron app

### ISSUE THREAD

**Issue by StenniHub** [LaunchMenu/node-global-key-listener#19]
I have attempted to use this package inside an electron application in place of iohook, but I have an issue where it fails to add a listener as it looks for WinKeyServer.exe in the incorrect place.

I've noticed in the source code it says:
`const sPath = "../../bin/WinKeyServer.exe";`
and later:
`this.proc = execFile(Path.join(__dirname, sPath), { maxBuffer: Infinity });`

Checking the output of __dirname it seems to point to the dist_electron folder generated during build, and the path it attempts to find WinKeyServer then becomes relative to this.

Is there any known ways to fix the initialization of the WinKeyServer inside an electron app that works for both development and production?

**Comment by StenniHub** [comment]
The __dirname output I mentioned might be misleading, as it seems to work in development mode. I got it working with some modifications I made in a fork which allows the path to the server executable to be overridden.
I doubt this is the most elegant solution however, since I also use this package https://github.com/sindresorhus/trash which similarly to this also executes a binary file bundled with the npm package, but required no modifications on my part to get working in electron. So far I have not been able to figure out what it does differently to avoid this issue.
